test/framework/kogitoserviceutils.go

```diff
@@ -98,14 +99,36 @@ func GetCliFlags(persistence, events bool) []string {
 	return cliFlags
 }
 
-func newImageOrDefault(fullImage string, defaultImage string) v1alpha1.Image {
+func newImageOrDefault(fullImage string, defaultImageName string) v1alpha1.Image {
 	if len(fullImage) > 0 {
 		return framework.ConvertImageTagToImage(fullImage)
 	}
 
-	image := framework.ConvertImageTagToImage(defaultImage)
-	image.Tag = config.GetServicesImageVersion()
-	return image
+	if len(config.GetServicesImageRegistry()) > 0 || len(config.GetServicesImageNamespace()) > 0 || len(config.GetServicesImageVersion()) > 0 {
+		registry := config.GetServicesImageRegistry()
```

Comment:
same here you, you could create the image with data from config and update the fields if empty ?

Reply:
Okis, changed.